third_party/rust/proc_macro2/v1/BUILD.gn

```diff
@@ -15,23 +15,23 @@ cargo_crate("buildrs_support") {
   crate_name = "proc_macro2"
   epoch = "1"
   crate_type = "rlib"
-  crate_root = "//third_party/rust/chromium_crates_io/vendor/proc-macro2-1.0.78/src/lib.rs"
+  crate_root = "//third_party/rust/chromium_crates_io/vendor/proc-macro2-1.0.80/src/lib.rs"
```

Comment:
Can this file just import upstream's build like we do for the other reflection crates? e.g.
```gn
group("lib") {
  public_deps = [ "//third_party/rust/proc_macro2/v1:lib" ]
}
```

Reply:
they don't have a buildrs_support target because they don't have any build deps on it